library/src/main/java/com/danikula/videocache/file/Files.java

```diff
@@ -47,7 +47,7 @@ static void makeDir(File directory) throws IOException {
     static void setLastModifiedNow(File file) throws IOException {
         if (file.exists()) {
             long now = System.currentTimeMillis();
-            boolean modified = file.setLastModified(now); // on some devices (e.g. Nexus 5) doesn't work
+            boolean modified = file.setLastModified(now/1000*1000); // on some devices (e.g. Nexus 5) doesn't work
```

Comment:
I've tested it [here](https://github.com/danikula/AndroidVideoCache/blob/master/test/src/test/java/com/danikula/videocache/HttpProxyCacheServerTest.java) and [here](https://github.com/danikula/AndroidVideoCache/blob/master/test/src/test/java/com/danikula/videocache/file/DiskUsageTest.java) and [here](https://github.com/danikula/AndroidVideoCache/blob/master/test/src/test/java/com/danikula/videocache/file/FileCacheTest.java). Can you point to real problem or better write test to proof that my implementation is wrong ?